test/test_json.py

```diff
@@ -0,0 +1,15 @@
+import json
+import os
+
+HOMEDIR = os.path.expanduser('~')
+file_location = os.path.dirname(os.path.realpath(__file__))
+retriever_root_dir = os.path.abspath(os.path.join(file_location, os.pardir))
+
+def json_test_module():
```

Comment:
The word `test` needs to go first in the function name so that `pytest` will detect it. So either `test_json_module` or `test_valid_json` would be good names.